godtools/App/Features/Menu/Presentation/Account/Subviews/AccountActivityView/Subviews/AccountActivityBadgeView.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  AccountActivityBadgeView.swift
+//  godtools
+//
+//  Created by Rachael Skeath on 1/23/23.
+//  Copyright © 2023 Cru. All rights reserved.
+//
+
+import SwiftUI
+
+struct AccountActivityBadgeView: View {
+    
+    let badge: UserActivityBadgeDomainModel
+    
+    let lightGreyTextColor = Color(red: 203 / 255, green: 203 / 255, blue: 203 / 255)
+    
+    var body: some View {
+        
+        ZStack(alignment: .center) {
+            
+            badgeBackground(withShadow: badge.isEarned)
+            
+            VStack(alignment: .center, spacing: 7) {
+                
+                ZStack(alignment: .center) {
+                    
+                    Image("badge_background")
+                        .foregroundColor(badge.iconBackgroundColor)
+                    Image(badge.iconImageName)
+                        .foregroundColor(badge.iconForegroundColor)
+                }
+                
+                Text(badge.badgeText)
+                    .font(FontLibrary.sfProTextRegular.font(size: 10))
+                    .foregroundColor(badge.isEarned ? ColorPalette.gtGrey.color : lightGreyTextColor)
```

Comment:
I think the badge textColor could even be part of the UseCase rather than checking isEarned here.